bridge/contracts/Federation.sol

```diff
@@ -164,6 +230,42 @@ contract Federation is Initializable, UpgradableOwnable {
         return true;
     }
 
+  function acceptTransfer(
+    address originalTokenAddress,
+    address payable sender,
+    address payable receiver,
+    uint256 number,
+    bytes32 blockHash,
+    bytes32 transactionHash,
+    uint32 logIndex,
+    TokenType tokenType
+  ) internal {
+    if (tokenType == TokenType.NFT) {
+      require(bridgeNFT != INFTBridge(0), "Federation: Empty bridgeNFT");
+      bridgeNFT.acceptTransfer(
+        originalTokenAddress,
+        sender,
+        receiver,
+        number,
```

Comment:
number is actually tokenId for NFT?

Reply:
Yes, he wants to use the same vote function for both bridges, so instead of amount it was renamed to number, if the name it's not ok, maybe value could work.
I think that maybe using 2 different functions, for voting, but they use internally the same voting logic, and also different events.